tools/reboot-cache-instances.sh

```diff
@@ -43,8 +43,9 @@ echo "It has instance_id $instance_id"
 
 asg_name=$(aws autoscaling describe-auto-scaling-instances --instance-ids "$instance_id" | jq .AutoScalingInstances[0].AutoScalingGroupName -r)
 
+echo "Setting $instance_id to standby in ASG $asg_name"
 aws autoscaling enter-standby --instance-ids "$instance_id" \
-  --auto-scaling-group-name "$asg_name" --should-decrement-desired-capacity
+  --auto-scaling-group-name "$asg_name" --should-decrement-desired-capacity | jq .Activities[0].Cause -r
```

Comment:
Is there guaranteed to always be exactly one activity in the response? Or is it that we always want the first one?

Reply:
If you just wanted to disable pagination, there's a `--no-paginate` flag on `aws` you could use

Reply:
Interesting, I'll give that a try, thanks. 

We only have one activity in the responses at present, but I guess it's not guaranteed to remain that way.  This script does have a pretty small scope though, so hopefully it won't diverge too far from the reality of our infra without someone remembering that [rebooting cache boxes is a thing](https://docs.publishing.service.gov.uk/manual/alerts/rebooting-machines.html#rebooting-cache-machines-in-aws).